scripts/validate-projects.ps1

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+$ErrorActionPreference = 'Stop'
+
+& "$PSScriptRoot/set-env.ps1"
+
+
+function Build-One {
+  param(
+      $project
+  );
+
+  Write-Host "##[info]Building $project..."
+  dotnet build $project `
+      -c $Env:BUILD_CONFIGURATION `
+      -v $Env:BUILD_VERBOSITY `
+      /property:DefineConstants=$Env:ASSEMBLY_CONSTANTS `
+      /property:Version=$Env:ASSEMBLY_VERSION `
+      /property:QsharpDocsOutDir=$Env:DOCS_OUTDIR
+
+  $script:all_ok = ($LastExitCode -eq 0) -and $script:all_ok
+}
+
+# Validating all katas projects can be disables with the ENABLE_KATAS flag:
```

Comment:
This comment does not seem true any longer? It seems that we dropped `ENABLE_KATAS` flag altogether in favor of the new split between different build scripts